### PR TITLE
Rename utility functions to not start with 'test'; fix prediction test

### PR DESCRIPTION
**Pull request by acrellin**

### CODE REVIEW

**Comment by acrellin** [comment]
:green_heart:

File: cesium_app/tests/test_datasets.py [hunk 1]
```diff
@@ -39,8 +39,8 @@ def test_add_new_dataset(driver):
 
 def test_delete_dataset(driver):
     # Delete data set
-    with test_project() as p:
-        with test_dataset(p) as ds:
+    with create_test_project() as p:
```
Comment (bnaul) [inline cesium_app/tests/test_datasets.py]: You can apparently put multiple context managers on the same line:

```
with create_test_project() as p, create_test_dataset(p) as ds:
```

**Comment by acrellin** [comment]
@bnaul I vaguely remembering `p` not being in the scope yet at that point, thus separating them - I'll try it again though

**Comment by bnaul** [comment]
I thought that might be the case but I just tried it and it worked for me

**Comment by acrellin** [comment]
@bnaul You're right - I'll do it that way instead. The indentation will be much better :)

**Comment by acrellin** [comment]
@bnaul done

**Comment by bnaul** [comment]
This is all pretty minor so I'm gonna go ahead and merge, sorry @stefanv 😬

**Pull request merged**